diff_definitions.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Axis Communications AB.
+# For a full list of individual contributors, please see the commit history.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Context-sensitive diffing of Eiffel type definitions. Compares the
+currently available types with the ones from a specified base and
+prints diff commands. For example, if the current commit has added
+v4.3.0 of ActT you'll get the following output:
+
+diff -u definitions/EiffelActivityTriggeredEvent/4.2.0.yml definitions/EiffelActivityTriggeredEvent/4.3.0.yml
+diff -u schemas/EiffelActivityTriggeredEvent/4.2.0.json schemas/EiffelActivityTriggeredEvent/4.3.0.json
+
+By default, the base of the comparison is origin/master, but any commit
+reference can be given as an argument.
+"""
+
+import sys
+from pathlib import Path
+
+import versions
+
+
+def _main():
+    base = "origin/master"
+    if len(sys.argv) > 2:
+        print(f"Usage: python {sys.argv[0]} [ base ]")
+        sys.exit(-1)
+    elif len(sys.argv) == 2:
+        base = sys.argv[1]
+
+    earlier = versions.latest_in_gitref(base, ".", Path("definitions"))
+    current = versions.latest_in_dir(Path("definitions"))
```

Comment:
Not sure if 'view' is the correct term in the suggestion below, but for clarity I'd suggest to change to the following:
```suggestion
    highest_in_base = versions.latest_in_gitref(base, ".", Path("definitions"))
    highest_in_view = versions.latest_in_dir(Path("definitions"))
```

Reply:
I changed to `base_defs` and `workspace_defs`.